src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -191,9 +189,7 @@ public void setPhone(Phone phone) {
             return Optional.ofNullable(phone);
         }
 
-        public void setEmail(Email email) {
-            this.email = email;
-        }
+        public void setEmail(Email email) { this.email = email; }
```

Comment:
<< Do edit this file back to the original copy (since there wasn't an actual change in the code)